Makefile

```diff
@@ -1,6 +1,6 @@
 all: build
 
-build:
+build: clean
```

Comment:
Builds take a second of which `clean` is half the time. Is that fast enough... it doesn't bother me yet. I added it because there were times where I thought parts of the code should be rebuilt(after I had fiddled with it) but gradle told me that there had been no change. I'm not sure whether it matters but it works for me so far.

Reply:
I mean a second seems fast enough to me.  If it gets annoyingly long we can look into why gradle isn't rebuilding things.